.storybook/recipes/GlobalHeader/GlobalHeader.tsx

```diff
@@ -44,7 +44,6 @@ export const GlobalHeader = ({ className, ...other }: Props) => {
       <Button
         className={styles['global-header__menu-button']}
         variant="icon"
-        inverted={true}
```

Comment:
Removing this prop here has no effect on this button because the icon has `fill: currentColor` (inside the Icon component) and the `Header` has its color set to white.